extensions/wikia/ThemeDesigner/ThemeDesignerHelper.class.php

```diff
@@ -52,22 +54,22 @@ public static function isValidColor( $sColor ) {
 			'slateblue', 'slategray', 'snow', 'springgreen', 'steelblue', 'tan',
 			'teal', 'thistle', 'tomato', 'turquoise', 'violet', 'wheat', 'white',
 			'whitesmoke', 'yellow', 'yellowgreen'
-		);
+		];
 
-		$isColorName = in_array ( strtolower( $sColor ), $aColorArray );
+		$isColorName = in_array( strtolower( $sColor ), $aColorArray );
 
-		Wikia::log(__METHOD__, "JKU", "[{$sColor}] is not a valid color name");
+		Wikia::log( __METHOD__, "JKU", "[{$sColor}] is not a valid color name" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace or remove this deprecated Wikia::log logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AWikiaLogUsage)